x-pack/plugins/cases/common/index.ts

```diff
@@ -15,12 +15,27 @@
 // For example, constants below could eventually be in a "kbn-cases-constants" instead.
 // See: https://docs.elastic.dev/kibana-dev-docs/key-concepts/platform-intro#public-plugin-api
 
-export { CASES_URL, SECURITY_SOLUTION_OWNER } from './constants';
+export {
+  CASES_URL,
+  SECURITY_SOLUTION_OWNER,
+  CREATE_CASES_CAPABILITY,
+  DELETE_CASES_CAPABILITY,
+  PUSH_CASES_CAPABILITY,
+  READ_CASES_CAPABILITY,
+  UPDATE_CASES_CAPABILITY,
+} from './constants';
 
 export { CommentType, CaseStatuses, getCasesFromAlertsUrl, throwErrors } from './api';
 
-export type { Case, Ecs, CasesFeatures, CaseViewRefreshPropInterface } from './ui/types';
+export type {
+  Case,
+  Ecs,
+  CasesFeatures,
+  CaseViewRefreshPropInterface,
+  CasesPermissions,
+} from './ui/types';
 
 export { StatusAll } from './ui/types';
 
 export { getCreateConnectorUrl, getAllConnectorsUrl } from './utils/connectors_api';
+export { createUICapabilities } from './utils/capabilities';
```

Comment:
Why don't we export this from the cases UI client?

Reply:
Yes, sorry I meant the cases server client. My main concern is the bundle size. Do you know if exporting from the client affects the bundle size?